### DISCUSSION TITLE
Sidebar deployment counter wrongly caps @ 500

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#63445] [Repositories]
### Select Topic Area

Bug

### Body

For example, https://github.com/kudoai/chatgpt.js/deployments shows over 1,000 deployments:

![image](https://github.com/community/community/assets/10906554/7197af32-176f-4f1d-a416-3a357cba1fd6)

...but repo sidebar consistently shows 500:

![image](https://github.com/community/community/assets/10906554/d2c5b682-15ee-485a-bbee-6891249eda72)

**Comment by JamesMGreene** [comment]
Thank you for the feedback! I've filed a bug internally to take a look into that. 🙇🏻
